plugin_ide.ui/plugin.xml

```diff
@@ -179,6 +186,12 @@
 			sequence="M1+O">
 		</key>
 		<key
+			commandId="com.github.rustdt.ide.ui.commands.OpenType"
+			contextId="com.github.rustdt.ide.ui.Contexts.Editor"

```

Comment:
This binding currently only works if an editor is open. Changing the `contextId` did not help. Any idea how to solve that problem?